lib/rubocop/cop/rails/blank.rb

```diff
@@ -82,12 +82,12 @@ def on_send(node)
         def on_or(node)
           return unless cop_config['NilOrEmpty']
 
-          nil_or_empty?(node) do |variable1, variable2|
-            return unless variable1 == variable2
+          nil_or_empty?(node) do |var1, var2|
+            return unless var1 == var2
```

Comment:
I really didn't want to do this. But. For the Rails `blank?` and `present?` cops, "variable" or "var" are the best descriptions of what these values contain.  (I don't find variable anymore descriptive than var.)  Thus, `var1` and `var2` are `AllowedNames`.